CHANGELOG.md

```diff
@@ -4,6 +4,26 @@ This file documents all notable changes to the GEOS-Chem repository since versio
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased TBD]
+### Added
```

Comment:
Should you also include "Added the carbon simulation to integration tests" (or something similar)?

Reply:
This is now done in commit 4efc5182ffb9c2d685b33957a62319e4c4da3ef6